kedro-docker/features/docker.feature

```diff
@@ -15,22 +14,6 @@ Feature: Docker commands in new projects
     And A .dive-ci file should exist
     And A .dockerignore file should exist
 
-  Scenario: Execute docker init --with-spark
```

Comment:
These two are moved to the separate file that uses different starter (`pyspark-iris`) to actually check spark context behaviour